ibm/mas_devops/roles/ocs/defaults/main.yml

```diff
@@ -1,3 +1,9 @@
 ---
 # We need to know what the OCP_RELEASE is to know what channel to target the subscriptions at
-ocp_release: "{{ lookup('env', 'OCP_RELEASE') | default('4.10', True) }}"
+ocp_release: "{{ lookup('env', 'OCP_RELEASE') | default('4.12', True) }}"
```

Comment:
While you're making changes here could you fix (remove) this OCP_RELEASE stuff ...

```
oc get clusterversion version
NAME      VERSION   AVAILABLE   PROGRESSING   SINCE   STATUS
version   4.12.7    True        False         36h     Cluster version is 4.12.7
```

User should not need to tell us the cluster release ... we can easily determin it from the clusterversion resource